src/ParquetMsg.chpl

```diff
@@ -316,11 +292,9 @@ module ParquetMsg {
               fileErrorCount += 1;
             }
         }
-        // This is handled in the readFilesByName() function
-        var subdoms: [filedom] domain(1);
         var len: int;
-        var nSeg: int;
         len = + reduce sizes;
```

Comment:
I know this is existing code but is there a reason for the `len` declaration to be spread out over 2 lines?

Reply:
No, this is a fragment from the HDF5 code that this was based on, I will condense that into one line